Tsa2341/my-brand-api
# #m9W0OdCo apis test

### What does this PR do?

this PR tests article, query, and user Apis

### Description of Task to be completed?

- test article, query, and user Apis

### How should this be manually tested?

- clone this repo
- in the terminal run "npm run test"
-wait for it to test and note that all tests passed



- or use this [link](https://my-brand-apis.herokuapp.com/api-docs)

### What are the relevant pivotal tracker stories?

- [#m9W0OdCo](https://trello.com/c/m9W0OdCo/25-test-all-apis)

### Screenshots (if appropriate)

- 
![Screenshot (35)](https://user-images.githubusercontent.com/83913153/152395612-b1dedac9-111a-46b7-8966-67dce4211486.png)
